docs/topics/mta-cli-analyze-selected-provider.adoc

```diff
@@ -0,0 +1,38 @@
+:_newdoc-version: 2.18.3
+:_template-generated: 2024-07-25
+:_mod-docs-content-type: PROCEDURE
+
+[id="selecting-language-providers-for-analysis_{context}"]
+= Analyzing a multi-language application for the selected supported language provider
+
+When analyzing a multi-language application with {ProductFullName} {CLINameTitle}, you can explicitly set a supported language provider according to your application language to run the analysis for.
+
+
+.Prerequisites
+
+* You have the following software installed on your system:
+** Python
+** Skopeo
```

Comment:
Why are these prereqs here? We don't need Python, Skopeo, jq, or oc for this

Reply:
@dymurray  These were provided by the QE when I asked if some prerequisites are required. Will remove.